peertalk/PTChannel.m

```diff
@@ -214,7 +214,20 @@ - (void)connectToPort:(int)port overUSBHub:(PTUSBHub*)usbHub deviceID:(NSNumber*
 }
 
 
-- (void)connectToPort:(in_port_t)port IPv4Address:(in_addr_t)ipv4Address callback:(void(^)(NSError *error, PTAddress *address))callback {
+- (void)connectToPort:(uint16_t)port IPv4Address:(NSString*)address callback:(void(^)(NSError *error, PTAddress *address))callback {
```

Comment:
The dns resolution point is valid and I'll accept that as a reason to revert to the parameter being a `in_addr_t`. I made this change because the `INADDR_LOOPBACK` isn't exposed to a swift codebase. I then spent to many hours trying to figure out why the address I generated with `inet_aton()` wasn't working. Come to find out it was because the `in_addr_t` returned by `inet_aton()` is already in translated to network byte order. My main question would be, Why on earth is the `INADDR_LOOPBACK` value not in network byte order already?

I suppose this issue of an `in_addr_t`'s byte order being un-discoverable and un-assertable can be solved with documenting the exposed interface. But then an inconsistency is introduced upon the user, "Why do I pass the port in machine byte order and the addr in network byte order?"

Documenting that we don't do DNS resolution and ONLY accept input the user can't screw up and is consistent with the other parameters is better then allowing the user to pass values with potentially opaque undefined behavior. IA possibility hoisting the user facing method even higher, where it does DNS resolution and it accepts a somewhat standard address port string(`"127.0.0.1:6000"`). If we do this then we remove access to the internal API that's easier to screw up.